pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -318,7 +318,11 @@ def choose_vcf_entry(vcf_entries, donor_unique_id, annotations):
                                     .format(donor_unique_id, workflow_previous.get('gnos_id'), '|'.join(workflow_previous.get('gnos_repo')))) 
             
             else:
-                if current_vcf_entry['is_oct2015_entry']:
+                if LooseVersion(current_vcf_entry.get('vcf_workflow_result_version')) > LooseVersion(workflow_previous.get('vcf_workflow_result_version')):
```

Comment:
I really like this here :+1: 
This works now, will work for ever without code change!